Automattic/sensei
# Disable Lesson creation when a course is created

Resolves #6979


## Proposed Changes
* Update all course patterns to remove the automatic lesson creation
* Update the container block to use more horizontal spacing and on Long Sales and Life Coach pattern

The idea of this PR is to disable the lesson's automatic creation to display the course outline screen and introduce AI.

NOTE: The new course outline screen cannot be displayed as the implementation PR (#6974) has not been merged yet.

## Testing Instructions

1. Enable the `ai_course_outline` feature flag.
2. Create a course
3. Choose a course pattern
4. Check if we are displaying an empty Course outline block instead of the default lessons.
5. Check all course pattern templates
6. Check if the feature toggle is off


## Pre-Merge Checklist

- [ ] Acceptance criteria is met (please check it)
